_episodes/08-kokkos-gpu.md

```diff
@@ -1,13 +1,15 @@
 ---
 title: "Kokkos with GPUs"
-teaching: 0
-exercises: 0
+teaching: 15
+exercises: 45
```

Comment:
I think we can reduce the complexity of the exercises here to reduce this time (rather than ask them to run on 1-5 nodes, let's just run on a single node for the comparison)

Reply:
Ditto